src/components/WelcomeMessage.tsx

```diff
@@ -0,0 +1,8 @@
+const WelcomeMessage = (): JSX.Element => {
+    return (
+        <span>Hello, Name!</span>
```

Comment:
Can you make 'Name' a prop here?

That way we can pass in any name from the `Room.tsx` component instead of hard coding it here